src/middlewares/clientAuthMiddleware.js

```diff
@@ -0,0 +1,77 @@
+import client from 'Config/client'; // eslint-disable-line import/no-unresolved
```

Comment:
@jpaylaga Is this a config containing the list of client as hash? If so maybe do a PR as well under https://github.com/reflex-media/lesgo/pulls for a sample. I suggest making this a reference: https://reflex-media.github.io/lesgo-docs/0.7.7/basics/middleware/

![image](https://user-images.githubusercontent.com/4793838/174249515-ed2ff174-f6fb-476d-b82f-379d7db8dd9e.png)

Reply:
Thanks @xXLXx but unsure if its a good idea to include the client config by default in lesgo repo as the properties directly under the client object is highly customized, unlike other configs available wherein their properties are fixed. Perhaps, changing the client config logic itself wherein object properties are fixed could be done, but it may take more time to do it. What do you think?

Reply:
Ahh hmm @jpaylaga so correct me if I'm wrong. I'm thinking that clientAuthMiddleware could be used for example in Vysion assume I have 1 PUT endpoint and another GET endpoint both accessed by some client platform that I've provided a key for, because say they have a subscription with us.

So in this case, I'm liking that fact that we should create a global config somehow (and that works out of the box), where we can just say that
`handler/put.ts` -> uses -> `Config/clientAuth.ts`
`handler/get.ts` -> uses -> `Config/clientAuth.ts`

and for instance we had a change of mind at that point we can just make use of the opts passed into the actual middleware like you said for highly customizability.